app/javascript/components/map/components/map-controls/map-controls-component.jsx

```diff
@@ -41,15 +66,34 @@ class MapControlsButtons extends PureComponent {
               onClick={() => toogleRecentImagery()}
               tooltip={{ text: 'Recent Imagery' }}
             >
-              <Icon icon={sateliteIcon} className="satelite-icon" />
+              <Icon icon={satelliteIcon} className="satelite-icon" />
             </Button>
-            <Button
-              className="basemaps-btn"
-              theme="theme-button-map-control"
-              tooltip={{ text: 'Basemaps' }}
+            <Tooltip
+              theme="light"
+              position="top-end"
+              useContext
+              interactive
+              open={showBasemaps}
+              onRequestClose={this.onTooltipRequestClose}
+              html={
+                <Basemaps
+                  onClose={this.toggleBasemaps}
+                  ref={this.setBasemapsRef}
+                />
+              }
             >
-              <Icon icon={globeIcon} className="globe-icon" />
-            </Button>
+              <Button
+                className="basemaps-btn"
+                theme="theme-button-map-control"
+                onClick={this.toggleBasemaps}
+                tooltip={{ text: 'Basemaps', hideOnClick: false }}
```

Comment:
Could we disable this when the basemaps window is open so that the user doesnt keep seeing the tooltip when they are viewing the basemaps?

Reply:
will do